app/views/content/is-teaching-right-for-me/engineers-teach-physics/_article.html.erb

```diff
@@ -50,7 +50,7 @@
   </section>
 
   <section class="col col-720 col-space-m">
-    <h2 class="heading-s">Engineers teach physics training programme</h2>
+    <h2>Engineers teach physics training programme</h2>
 
     <p>This new programme is based on the physics initial teacher training (ITT) course, and has been developed with support from the physics and engineering community. It has been adapted to help you make the transition from engineering or material sciences to teaching physics. The course is designed to capitalise on your experience and empower you with the skills needed to become a great physics teacher.
      </p>
```

Comment:
@caitlin-reidy In the bit after this it says: Your training provider may offer part-time teacher training courses and may also allow you to combine courses, for example Engineers teach physics combined with mathematics.

I wasn't sure if that's the name of a course Engineers teach physics combined with mathematics or if it's Engineers teach physics combined with mathematics as in two separate course names in which case should Mathematics be capped so they're consistent?